### PR TITLE
Spelling: Avoiding "translation keys". and "units".

### PR DESCRIPTION
**Pull request by comradekingu**
A lot of guesswork going on here.

### CODE REVIEW

File: weblate/addons/resx.py [hunk 1]
```diff
@@ -31,8 +31,8 @@ class ResxUpdateAddon(BaseCleanupAddon):
     verbose = _("Update RESX files")
     description = _(
         "Update all translation files to match the monolingual base file. "
-        "It removes stale translation keys and adds new ones with source "
-        "string as a translation."
+        "It removes unused strings, and adds new ones wherein the translation is "
```
Comment (comradekingu) [inline weblate/addons/resx.py]: Removes from where, unused as in?

Comment (nijel) [reply weblate/addons/resx.py]: Short summary: It does pretty much same thing as msgmerge does for gettext PO files

It takes the source language file and removes from target language files any strings which are no longer present there, adds new ones and updates changed ones. This used to be part of the cleanup addon, but it turned out that only part of RESX users want the updating part, it depends how RESX is implemented in the application.

File: weblate/addons/resx.py [hunk 2]
```diff
@@ -67,7 +67,7 @@ def update_resx(self, index, translation, storage, changes):
 
     @staticmethod
     def find_changes(index, storage):
-        """Find changed units in storage"""
+        """Find changed string IDs in upstream repository"""
```
Comment (nijel) [inline weblate/addons/resx.py]: In the code, we still call these units. Also it's not a repository, it's a file storage.

Comment (comradekingu) [reply weblate/addons/resx.py]: The difference is lost on me here. Does it just find them, or something more? How about "Find changed string IDs upstream"?

Comment (nijel) [reply weblate/addons/resx.py]: Unit is an object representation of what we call string in the interface. Mostly for historical reasons it's called differently in the code (actually it matches translate-toolkit terminology here).

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**